.github/workflows/unit_tests.yml

```diff
@@ -98,7 +98,8 @@ jobs:
       - name: Install build tools
         run: |
           ${{ matrix.install }}
-          sudo apt install -y ninja-build
+          sudo apt install -y ninja-build python3-pip
+          sudo pip3 install --break-system-packages pytest pytest-forked hypothesis
```

Comment:
**issue**: Using pip isn't right for ubuntu 24.04 - need to use pipx instead.

Reply:
See https://github.com/intel/cpp-std-extensions/blob/main/.github/workflows/unit_tests.yml#L234 for example